manifests/eth.pp

```diff
@@ -144,51 +145,55 @@
   Optional[Boolean]                 $ipv6_control_radvd               = undef,
   Optional[Integer]                 $ipv6_mtu                         = undef,
-  String                            $ipv6_privacy                     = 'rfc3041',
-  Optional[String]                  $ipv6_radvd_pidfile               = undef,
-  Optional[String]                  $ipv6_radvd_trigger_action        = undef,
+  String[1]                         $ipv6_privacy                     = 'rfc3041',
+  Optional[String[1]]               $ipv6_radvd_pidfile               = undef,
+  Optional[String[1]]               $ipv6_radvd_trigger_action        = undef,
   Optional[Boolean]                 $ipv6_router                      = undef,
   Optional[Simplib::IP::V6]         $ipv6addr                         = undef,
   Optional[Array[Simplib::IP::V6]]  $ipv6addr_secondaries             = undef,
   Optional[Boolean]                 $ipv6init                         = undef,
-  Optional[String]                  $ipv6to4_ipv4addr                 = undef,
+  Optional[String[1]]               $ipv6to4_ipv4addr                 = undef,
   Optional[Integer]                 $ipv6to4_mtu                      = undef,
-  Optional[String]                  $ipv6to4_relay                    = undef,
-  Optional[String]                  $ipv6to4_routing                  = undef,
+  Optional[String[1]]               $ipv6to4_relay                    = undef,
+  Optional[String[1]]               $ipv6to4_routing                  = undef,
   Optional[Boolean]                 $ipv6to4init                      = undef,
   Optional[Boolean]                 $isalias                          = undef,
-  Optional[Integer]                 $linkdelay                        = undef,
+  Optional[Integer[1]]              $linkdelay                        = undef,
   Optional[Simplib::Macaddress]     $macaddr                          = undef,
-  Optional[String]                  $master                           = undef,
-  Optional[String]                  $metric                           = undef,
-  Optional[Integer]                 $mtu                              = undef,
-  Optional[String]                  $net_type                         = undef,
+  Optional[String[1]]               $master                           = undef,
+  Optional[String[1]]               $metric                           = undef,
+  Optional[Integer[1]]              $mtu                              = undef,
+  Optional[String[1]]               $net_type                         = undef,
   Optional[Simplib::IP]             $netmask                          = undef,
   Optional[Simplib::IP]             $network                          = undef,
-  Optional[String]                  $nozeroconf                       = undef,
+  Boolean                           $nm_controlled                    = pick(fact('simplib_networkmanager.enabled'), false),
```

Comment:
Minimum version of simplib in metadata.json should be updated to the version of simplib that provides this fact

Reply:
:+1:

Reply:
fixed